docs/DESIGN.md

```diff
@@ -21,30 +21,30 @@ available in `rust-vmm`:
 [`event-manager`](https://crates.io/crates/event-manager),
 [`vm-superio`](https://crates.io/crates/vm-superio) and
 [`vmm-sys-util`](https://crates.io/crates/vmm-sys-util). Only the serial
-console device will be supported. The next iteration will add virtio devices
-and vCPU abstractions. Without the latter, vCPU configuration will be handled
-directly through `kvm-ioctls` and will not be abstracted through interfaces.
+console device is supported. The current iteration is adding virtio devices
+and vCPU abstractions.
 
 Complex functionalities that require ongoing user interaction with the VMM are
-not supported in this version. Such functionalities include but are not
-limited to: device hotplug, VM pause/resume, snapshotting, migration. As the
+not currently supported. Such functionalities include but are not limited to:
+device hotplug, VM pause/resume, snapshotting, migration. Next step is adding
+the VM pause/resume support, see
+[tracking issue](https://github.com/rust-vmm/vmm-reference/issues/58). As the
 necessary building blocks emerge in `rust-vmm`, integration in the `vmm` crate
 will not pose a large effort, but the frontend / app will require extensive
-modifications.
+modifications, see
+[tracking issue](https://github.com/rust-vmm/vmm-reference/issues/55).
 
 ## Overview
 
 The reference VMM is composed of 2 major crates packaged in a binary
-application: the `vmm` and a oneshot CLI that invokes it. The `vmm` crate
+application: the `vmm` and an oneshot CLI that invokes it. The `vmm` crate
```

Comment:
nit: `a oneshot` is correct here, pronunciation-wise `oneshot` *sounds* as if it started with the consonant `w` (see [a more compelling explanation](https://www.grammar.com/a-vs-an-when-to-use))

Reply:
:O Just followed the IDE suggestion :(. I'll revert, thanks!